rfc6962-bis.xml

```diff
@@ -1656,13 +1674,12 @@ this can be done.
 Merkle inclusion proof (<xref target="get-proof-by-hash"/>) for each observed SCT.
 These checks can be asynchronous and
 need only be done once per each certificate. In order to protect the clients'
-privacy, these checks need not reveal the exact certificate to the log. Clients
-can instead request the proof from a trusted auditor (since anyone can compute
-the proofs from the log) or request Merkle inclusion proofs for a batch of
-certificates around the SCT timestamp.
+privacy, these checks need not reveal the exact certificate to the log. Instead, clients
+can request the proof from a trusted auditor (since anyone can compute
+the proofs from the log) or communicate with the log via proxies.
         </t>
         <t>
-          Violation of the append-only property can be detected by clients comparing their instances of the Signed Tree Heads. As soon as two conflicting Signed Tree Heads for the same log are detected, this is cryptographic proof of that log's misbehavior. There are various ways this could be done, for example via gossip (see <xref target="I-D.ietf-trans-gossip"/>) or peer-to-peer communications or by sending STHs to monitors (who could then directly check against their own copy of the relevant log).
+          Violation of the append-only property or the STH issuance rate limit can be detected by clients comparing their instances of the Signed Tree Heads. As soon as two conflicting Signed Tree Heads for the same log are detected, this is cryptographic proof of that log's misbehavior. There are various ways this could be done, for example via gossip (see <xref target="I-D.ietf-trans-gossip"/>) or peer-to-peer communications or by sending STHs to monitors (who could then directly check against their own copy of the relevant log).
```

Comment:
This is a little bit of an oversimplifcation. The proof of misbehaviour is that one of the STHs doesn't match a calculated STH from a copy of the log. You never get a proof of inconsistency between two STHs, you just can't get a proof of consistency, so a log, when asked for one, will presumably not respond, or respond with an incorrect proof (which still doesn't prove anything).